packages/frontend/app/(dashboard)/profile/components/AvatarSettings.tsx

```diff
@@ -15,59 +15,14 @@ const AvatarSettings: React.FC = () => {
   const isMobile = useMediaQuery('(max-width: 768px)')
   const [avatarSize, setAvatarSize] = useState(windowWidth / 2)
   const [uploading, setUploading] = useState(false)
+  const [isUploadModalOpen, setIsUploadModalOpen] = useState(false)
   const { userInfo, setUserInfo } = useUserInfo()
 
-  const { data: profile, mutate } = useSWR(`api/v1/profile`, async () =>
-    API.profile.index(),
-  )
-
   useEffect(() => {
     const widthDivider = isMobile ? 3 : 10
     setAvatarSize(windowWidth / widthDivider)
   }, [windowWidth, isMobile])
 
-  const beforeUpload = (file: any) => {
-    const isJpgOrPng = file.type === 'image/jpeg' || file.type === 'image/png'
-
-    if (!isJpgOrPng) {
-      message.error('You can only upload JPGs or PNGs!')
-    }
-
-    const isLt1M = file.size / 1024 / 1024 < 1
-    if (!isLt1M) {
-      message.error('Image must smaller than 1MB!')
-    }
-
-    return isJpgOrPng && isLt1M
-  }
-
-  const handleUpload = async (file: any) => {
```

Comment:
i like that you standardized the photo upload

Reply:
Thanks!